package.json

```diff
@@ -53,7 +53,7 @@
     "path-to-regexp": "^1.7.0",
     "prop-types": "^15.5.10",
     "react-native-drawer-layout-polyfill": "^1.3.2",
-    "react-native-tab-view": "^0.0.69"
+    "react-native-tab-view": "^0.0.70"
```

Comment:
Has this been checked to make sure the breaking changes in this release didn't break any of the tab logic in the app?

Reply:
Just checked the code and looks like we're in the clear 👍

Reply:
To give you some context, it was a [necessary update](https://github.com/react-community/react-navigation/issues/2771#issuecomment-339034027) ;) And yeah it works 👌